### ISSUE TITLE
Component does not destroy Popper.js reference when it's removed

### ISSUE THREAD

**Issue by absoike** [snowcoders/react-popper#8]
Need to define componentWillUnmount like [react-popper](https://github.com/FezVrasta/react-popper/blob/master/src/Popper.js)
```
  componentWillUnmount() {
    if (this.state.popperInstance) {
      this.state.popperInstance.destroy();
    }
  }
```
To prevent this error
<img width="891" alt="screen shot 2018-05-29 at 1 35 15 pm" src="https://user-images.githubusercontent.com/4752131/40684566-812126d0-6346-11e8-978b-7a0eaffafe5a.png">

**Comment by k2snowman69** [comment]
Cool! Looks like this was also fixed in the FezVrasta version (which is in beta) a bit ago. Just merged the pull request, should have a new npm build out soon. 

https://github.com/snowcoders/react-popper/pull/9

**Comment by k2snowman69** [comment]
Should be resolved with 0.9.2
